src/cryptography/hazmat/primitives/serialization/ssh.py

```diff
@@ -572,13 +576,58 @@ def encode_private(
         f_priv.put_sshstr(f_keypair)
 
 
+class _SSHFormatSK:
+    def load_application(self, data) -> tuple[memoryview, memoryview]:
+        application, data = _get_sshstr(data)
```

Comment:
Given that we don't use the application, what do you think about raising an exception if it's not `ssh://`. We can later loosen this if it's necessary.

Reply:
done + tests